src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -206,7 +209,12 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
 
     // Fetch all the saved objects that exist so we can accurately populate the counts within
     // the table filter dropdown.
-    const savedObjectCounts = await getSavedObjectCounts(this.props.http, allowedTypes, queryText);
+    const savedObjectCounts = await getSavedObjectCounts(
+      this.props.http,
+      allowedTypes,
+      [],
```

Comment:
Why passing empty array here, base on the context seems here should pass in the allowedNameSpaces?

Reply:
Hey Kristen, thank you for pointing this out. For this implementation I was thinking that since namespaces are registered by plugins and not a static amount like with `types`, that passing an empty array would indicate to the API to fetch objects from all namespaces and return a count of the namespaces associated with the fetched objects. That's different than how the `types` works though where it assumes empty array means fetch no objects.

FYI I am working on a follow-up to this PR with a fix for a regression introduced on exporting button because of the `filteredSavedObjectCounts` type above since objects counts will now be a data structure like 

```
{ 
  'types': {
     'config': 6,
     ...,
  },
  'namespaces': {
    'user_tenant': 5,
   ...
  }
}
```

instead of:

```
{
  'config': 5,
  'index-pattern': 7,
  ...
}
```

I have some changes ready, but want to add better test coverage. You can see changes being prepared here: https://github.com/opensearch-project/OpenSearch-Dashboards/compare/main...cwperks:OpenSearch-Dashboards:namespace-registry-fixes